cmake/ETLSetupFeatures.cmake

```diff
@@ -37,18 +65,18 @@ if(BUILD_CLIENT)
 		find_package(OpenGL REQUIRED)
 		list(APPEND RENDERER_LIBRARIES ${OPENGL_LIBRARIES})
 		include_directories(SYSTEM ${OPENGL_INCLUDE_DIR})
-	else() # FEATURE_RENDERER_GLES
+	else() # FEATURE_RENDERER_GLES	
 		find_package(GLES REQUIRED)
 		list(APPEND RENDERER_LIBRARIES ${GLES_LIBRARY})
 		include_directories(SYSTEM ${GLES_INCLUDE_DIR})
 	endif()
 
 	if(NOT BUNDLED_SDL)
-		# specials for ARM - currently set for RPI raspdian/ubuntu mate
-		# system sdl lib isn't latest & sndio lib is required
+		# specials for ARM - currently set for RPI Raspbian/ubuntu mate
+		# system sdl lib isn't latest & sndio lib is required			
```

Comment:
Is this comment still up-to-date?
It seems the `if(ARM)` condition below isn't required anymore, since both branch are identical now.

Reply:
Well spotted. Doesn't seem to be required. Will update.